### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by stephantual**
Updated Readme contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,105 @@
...
 
-While the Service Provider is bound by the term of the proposal, the DAO participants can elect to ‘pull the plug’ on the Service Provider at anytime if they feel they are not getting their money’s worth. This is a major advantage over the Kickstarter or 'pre-sale' models as it considerably minimizes risk. 
+## Disclaimer
 
-It is also possible for the DAO to elect a replacement Service Provider, meaning that the project can continue where it left off. In essence, by decoupling finance from operations this DAO model survives where other means of funding would have failed.
+The future remains a work in progress. Our vision exists in a world where laws vary widely. It is important to remember that anyone who uses DAO code will do so at their own risk. We can’t speculate about the legal status of DAOs worldwide. Whatever our personal beliefs may be, people must draw their own conclusions, relying on legal advice where appropriate. The Authors are not law firm and we are not in the business of offering legal advice.
```
Comment (LefterisJP) [inline README.md]: `The Authors are not law firm` -> The Authors are not a law firm`

File: README.md [hunk 2]
```diff
@@ -1,65 +1,105 @@
...
+
+
+
+
+
+## Overview
+
+Our generic DAO code allows people to create self-governing, decentralized autonomous organizations governed by software code with instructions written immutably to the blockchain.
+
+We are making the generic DAO model we developed free and open source, so it can be reused by anyone wishing to put together a transparent organization where governance and decision making system are immutably programmed in the Blockchain. This code been reviewed by hundreds of pairs of eyes from our community and by one of the most respected auditing company in the world, Deja Vu.
```
Comment (LefterisJP) [inline README.md]: system -> systems

Comment (LefterisJP) [inline README.md]: company -> companies

File: README.md [hunk 3]
```diff
@@ -1,65 +1,105 @@
...
+
+This DAO model is open source under the LGPL, so it can be reused by anyone wishing to put together a transparent organization where governance and decision making system are immutably programmed in the Blockchain.
+
+Note: Although the word "contract" is used in the DAO code, the term is a programming convention and is not being used as a legal term of art.  The DAO does not practice law and the authors of this code do not practice law are not your lawyers.  The term is a programming convention, not a representation that the code is in and of itself a legally binding and enforceable contract. If you have questions about legal enforceability, consult with legal counsel.
```
Comment (LefterisJP) [inline README.md]: `do not practice law are not your` -> `do not practice law and are not your`

File: README.md [hunk 4]
```diff
@@ -1,65 +1,105 @@
...
 ### Token.sol: 
 Defines the functions to check token balances, send tokens, send tokens on behalf of a 3rd party and its corresponding approval process.
 
 ### TokenSale.sol: 
-Token Sale contract, used by the DAO to raise its funding goal
+Token Sale contract, used by the DAO to sell its tokens and initialize its ether.
 
 ### SampleOffer.sol
-Sample Proposal from a Service Provider to the DAO
+Sample Proposal from a Service Provider to the DAO. Feel free to use as a template for your own proposal.
```
Comment (LefterisJP) [inline README.md]: Shouldn't this be Contractor?

File: README.md [hunk 5]
```diff
@@ -1,65 +1,105 @@
...
 
 ### ManagedAccount.sol
-Basic account, managed by another contract
+Basic account, used by the DAO contract to separately manage both the rewards and the extraBalance accounts.
```
Comment (LefterisJP) [inline README.md]: `Basic account` -> `Basic managed account`

File: README.md (hunk 2 above)
Comment (stephantual) [inline README.md]: actually company looks correct to me

Comment (LefterisJP) [inline README.md]: no trust me, it should be companies.

Comment (LefterisJP) [inline README.md]: Also `This code been` -> `This code has been`

Comment (stephantual) [inline README.md]: you're correct. I'll update this and the blog post it came from

**Comment by LefterisJP** [comment]
Superseded by: https://github.com/slockit/DAO/pull/92

**Pull request closed without merging**